internal/logger/logger.go

```diff
@@ -173,7 +208,8 @@ type loggerFactory struct {
 func (f *loggerFactory) newLogger(level string) *slog.Logger {
 	// create a new logger
 	var programLevel = new(slog.LevelVar)
-	logger := slog.New(f.handler(programLevel, ""))
+	logger := slog.New(f.handler(programLevel, "").WithAttrs([]slog.Attr{slog.String(mountLoggerIdKey,
+		fmt.Sprintf("%s-%s", MountFsName, MountInstanceId))}))
```

Comment:
I'm not sure about the performance characteristics of using Sprintf here. Should we do something like:
```suggestion
	logger := slog.New(f.handler(programLevel, "").WithAttrs([]slog.Attr{slog.String(mountLoggerIdKey, MountInstanceId), slog.String("fs", MountFsName))}))
```

Reply:
Makes sense. Thanks for explaining.

I'd also like to understand why did you choose to log one attribute instead of two: mount-id and bucket separately?

Secondly, how do you plan to assess the performance characteristics of this change?